install_config/aggregate_logging.adoc

```diff
@@ -1343,6 +1343,206 @@ This causes Fluentd to write logs to the Fluentd default location, *_/var/log/fl
 oc set env ds/fluentd LOGGING_FILE_PATH=console
 ----
 
+[[fluentd-merge-json]]
+*Configuring log collection JSON parsing*
+
+You can configure the Fluentd log collector to determine if a log message is in *JSON* format and merge
+the message into the JSON payload document posted to Elasticsearch. This feature is disabled by default.
+
+You can enable or disable this feature by editing the `MERGE_JSON_LOG` environment variable in the *fluentd* daemonset.
+
+[IMPORTANT]
+====
+Enabling this feature comes with risks, including:
+
+* Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings.
+* Potential buffer storage leak caused by rejected message cycling.
+* Overwrite of data for field with same names.
+
+The features in this topic should be used by only experienced Fluentd and Elasticsearch users.
+====
+
+To enable this feature:
+
+. Run the following command:
++
+----
+oc set env ds/logging-fluentd MERGE_JSON_LOG=true <1>
+----
+<1> Set this to `false` to disable this feature or `true` to enable this feature.
```

Comment:
"If you want to make sure this setting is applied every time you run ansible, add `openshift_logging_fluentd_merge_json_log="false"` to your ansible inventory."